lib/modules/manager/custom/jsonata/index.ts

```diff
@@ -42,12 +43,28 @@ export async function extractPackageFile(
     return null;
   }
 
-  const deps = await handleMatching(json, packageFile, config);
-  if (!deps.length) {
-    return null;
+  let deps = await handleMatching(json, packageFile, config);
+
+  // filter all null values
+  deps = deps.filter(is.truthy);
```

Comment:
isn't that already done in the handle function? 🤔

Reply:
True